code/game/machinery/adv_med.dm

```diff
@@ -410,7 +410,7 @@
 			visible_message("<span class='notice'>\The [src] rattles and prints out a sheet of paper.</span>")
 			var/obj/item/weapon/paper/P = new /obj/item/weapon/paper(loc)
 			P.info = "<CENTER><B>Body Scan - [href_list["name"]]</B></CENTER><BR>"
-			P.info += "<b>Time of scan:</b> [worldtime2text(world.time)]<br><br>"
+			P.info += "<b>Time of scan:</b> [stationtime2text(world.time)]<br><br>"
```

Comment:
All lines that say ```stationtime2text(x)``` should likely be ```station_adjusted_time(x)```. You'll note that the proc doesn't actually take or do anything with arguments.
It's something I need to correct for Bay.